mw-webapp/src/pages/authPage/AuthPage.tsx

```diff
@@ -1,14 +1,62 @@
-import {Link} from "react-router-dom";
+import {User, getRedirectResult, onAuthStateChanged} from "firebase/auth";
+import {useEffect, useState} from "react";
+import {Link, useNavigate} from "react-router-dom";
 import {useAuth} from "src/utils/useAuth";
 
 export const AuthPage = () => {
-  const handleGoogleSignIn = useAuth();
+  const {handleGoogleSignIn, handleLogout, writeNewUserData, auth} = useAuth();
```

Comment:
Is it possible to rename this consts or it's naming from firebase? For make it more similar - for example to change handleGoogleSignIn -> handleLogIn?

Reply:
Done